tools/tokenserver/loadtests/README.md

```diff
@@ -4,37 +4,68 @@ This directory contains everything needed to run the suite of load tests for Tok
 
 ## Building and Running
 1. Install the load testing dependencies:
-```sh
-pip3 install -r requirements.txt
-```
-2. Set up a mock OAuth verifier, with which Tokenserver will verify OAuth tokens. The subdirectory [mock-fxa-server/](./mock-fxa-server) includes code deployable as a GCP Cloud Function that acts as a mock FxA server, "verifying" OAuth tokens. You can deploy your own Cloud Function by running the following command in this directory:
-```sh
-gcloud functions deploy mock_fxa_server --runtime=python39 --trigger-http --source=mock-fxa-server
-```
-You can stand up a local copy of the Cloud Function by running the following in this directory:
-```sh
-functions-framework --target mock_fxa_server --debug
-```
-Note that you'll need to install `functions-framework` via `pip3 install functions-framework`. The load tests will use FxA stage to verify BrowserID assertions.
-3. Configure Tokenserver to verify OAuth tokens through the mock FxA service and BrowserID assertions through FxA stage. This is done by setting the following environment variables:
-```
-SYNC_TOKENSERVER__FXA_BROWSERID_AUDIENCE=https://token.stage.mozaws.net
-SYNC_TOKENSERVER__FXA_BROWSERID_ISSUER=api-accounts.stage.mozaws.net
-SYNC_TOKENSERVER__BROWSERID_VERIFIER_URL=https://verifier.stage.mozaws.net/v2
+    ```sh
```

Comment:
Super minor nit, but I wonder if we should run the `.md` files through `prettier` or something. There are a lot of formatting warnings about no empty lines around headers and code blocks. Totally fine if we don't.

Reply:
Are you thinking we should run `prettier` in CI?

Reply:
I don't know if we modify the file enough to warrant that, nor do I feel like it's a blocker. We might want to run the files through at least once so that they're consistent.